orttraining/orttraining/test/python/orttraining_test_orttrainer_bert_toy_onnx.py

```diff
@@ -17,59 +19,518 @@
+    (amp.DynamicLossScaler()),
+    (CustomLossScaler())
+])
+def testToyBERTModelMixedPrecisionLossScaler(loss_scaler):
+    total_steps = 10
+
+    model_desc = bert_model_description()
+    model = load_bert_onnx_model()
+
+    optim_config = optim.LambConfig()
+    opts =  orttrainer.ORTTrainerOptions({
+        'debug' : {
+            'deterministic_compute': True
+        },
+        'mixed_precision': {
+            'enabled': True,
+            'loss_scaler': loss_scaler
+        }
+    })
+    
+    # Instantiate ORTTrainer
+    trainer = orttrainer.ORTTrainer(model, model_desc, optim_config, options=opts)
+    
+    losses = []
+    for i in range(total_steps):
+        sample_input = generate_random_input_from_model_desc(model_desc, i)
+
+        losses.append(trainer.train_step(*sample_input).cpu().item())
+
+
+###############################################################################
+# Temporary tests comparing Legacy vs Experimental ORTTrainer APIs ############
+###############################################################################
+
+
+def testToyBERTModelLegacyExperimental():
+    num_batches = 10
+
+    model_desc = bert_model_description()
+    model = load_bert_onnx_model()
+
+    params = optimizer_parameters(model)
+    #optim_config = optim.LambConfig(params, alpha= 0.9, beta= 0.999, lambda_coef= 0.01, epsilon= 1e-6)
+    optim_config = optim.LambConfig()
+    opts =  orttrainer.ORTTrainerOptions({
+        'debug' : {
+            'deterministic_compute': True
+        },
+    })
+    
+    torch.manual_seed(1)
+    set_seed(1)
+    # Instantiate ORTTrainer
+    trainer = orttrainer.ORTTrainer(model, model_desc, optim_config, options=opts)
+    
+    experimental_losses = []
+    for i in range(num_batches):
+        sample_input = generate_random_input_from_model_desc(model_desc, i)
+
+        experimental_losses.append(trainer.train_step(*sample_input).cpu().item())
+
+    # LEGACY IMPLEMENTATION
+    device = torch.device("cuda", 0)
+    legacy_model_desc, learning_rate_description, learning_rate = legacy_model_params() 
+    torch.manual_seed(1)
+    set_seed(1)
+    
+    legacy_trainer = Legacy_ORTTrainer(model, None, legacy_model_desc, "LambOptimizer",
+                       None,
+                       learning_rate_description,
+                       device)
+    legacy_losses = []
+    for i in range(num_batches):
+        sample_input = generate_random_input_from_model_desc(model_desc, i)
+        legacy_sample_input = [*sample_input, learning_rate]
+
+        legacy_losses.append(legacy_trainer.train_step(legacy_sample_input).cpu().item())
+
+    _test_helpers.assert_model_outputs(experimental_losses, legacy_losses, True, rtol=1e-4)
+
+@pytest.mark.parametrize("initial_lr, lr_scheduler, legacy_lr_scheduler", [
+    (1.0, optim.lr_scheduler.ConstantWarmupLRScheduler, constantlrscheduler),
+])
+def testToyBERTModelLRSchedulerLegacyExperimental(initial_lr, lr_scheduler, legacy_lr_scheduler):
```

Comment:
```suggestion
def testToyBERTModelLegacyExperimentalLRScheduler(initial_lr, lr_scheduler, legacy_lr_scheduler):
```

Reply:
The prefix must be identical between all tests to allow selective tests like `pytest -sk testToyBERTModelLegacyExperimental*`